packages/shared/friends/selectors.ts

```diff
@@ -38,29 +38,6 @@ export const getConversations = (
     }))
 }
 
-/**
- * Get all current conversations with messages the user has including DMs, channels, etc
- * @return `conversation` & `unreadMessages` boolean that indicates whether the conversation has unread messages.
- */
-export const getAllConversationsWithMessages = (
```

Comment:
I'm killing this one: https://github.com/decentraland/kernel/pull/649#discussion_r1008429971